test/parallel/test-http2-create-client-secure-session.js

```diff
@@ -5,15 +5,16 @@ const common = require('../common');
 if (!common.hasCrypto)
   common.skip('missing crypto');
 
+const fixtures = require('../common/fixtures');
+
 const assert = require('assert');
 const path = require('path');
 const fs = require('fs');
```

Comment:
We no longer need `fs` as it is no longer used, can you please remove it? The linter is not happy if we keep it :)

Reply:
Oh yea, missed that. Thanks!